<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>392</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Modbus Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>155</width>
     <height>267</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="title">
    <string>Serial</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Slave 1:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comselCombo_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QComboBox" name="parityCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comselCombo_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Slave 3:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QComboBox" name="baudCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comselCombo_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Stop Bits:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comselCombo_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Slave 4:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Data Bits:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="stopBitsCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Baud Rate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Slave 2:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comselCombo_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="comselCombo_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QComboBox" name="dataBitsCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Slave 6:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Slave 5:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="title">
    <string>TCP</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP addr:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port no. begin from:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="IntInputEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>62</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>502</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="ipCombo">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>28</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="IntInputEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Device Id begin from:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Connect:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="conselCombo">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>TCP</string>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Serial</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>166</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="applyButton">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cancelButton">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntInputEdit</class>
   <extends>QLineEdit</extends>
   <header>intinputedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>conselCombo</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>comselCombo_1</tabstop>
  <tabstop>baudCombo</tabstop>
  <tabstop>dataBitsCombo</tabstop>
  <tabstop>parityCombo</tabstop>
  <tabstop>stopBitsCombo</tabstop>
  <tabstop>portEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
